### DISCUSSION TITLE
Не работает "Своя вкладка"

### DISCUSSION THREAD

**Discussion by Korz431** [Chimildic/YaMuTools#97] [Помощь с ошибкой]
Перестала работать пользовательская вкладка, ранее отлично работала, но недавно она перестала отображаться хотя в настройках включена. Я пробовал уже включать и отключать её, менять название и редактировать ссылку, но ничего не помогло.

**Comment by KIRILL-VELES** [comment]
Оплатил ни чего не произошло, обраной связи нету,Препалагаю,что данная персона мошенник!

**Reply by Chimildic** [reply]
Весь функционал расширения бесплатен. Донат добровольный. Приложение для андроид существует отдельно. Сообщите куда и что вы отправляли, я верну ваши деньги.

Расширение с течением времени ломается из-за изменений на сайте. Я единственный разработчик и не могу вечно поддерживать его в актуальном состоянии.

**Reply by KIRILL-VELES** [reply]
Задумка с приложением не плохая,но если вы единственный разработчик и у вас нету времени на поддержку работоспособни, то возможно это плохая идея держать в открытом доступе и вводить людей в заблуждение. Я отправил Донат заранее и всегда поддерживаю ребят,но тут бездарность.

**Reply by Chimildic** [reply]
Ваш кошелёк *8966?
